bin/docker_push

```diff
@@ -10,7 +10,6 @@ echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdi
 docker tag "${TRAVIS_REPO_SLUG}:latest" "${DOCKER_IMAGE}"
 
 if [[ "${CI}" == "true" ]]; then
-    docker push "${TRAVIS_REPO_SLUG}:latest"
-    docker push "${DOCKER_IMAGE}"
+    docker push "${TRAVIS_REPO_SLUG}
```

Comment:
tsk tsk.. this got approved (and I merged) with a missing `"` here. I'll fix directly on master.

Reply:
Damn.